logstash-core/lib/logstash/logging/logger.rb

```diff
@@ -70,7 +72,16 @@ def self.configure_logging(level, path = LogManager::ROOT_LOGGER_NAME)
       def self.initialize(config_location)
         @@config_mutex.synchronize do
           if @@logging_context.nil?
-            @@logging_context = Configurator.initialize(nil, config_location)
+            file_path = URI(config_location).path
+            if ::File.exists?(file_path)
+              logs_location = java.lang.System.getProperty("ls.logs")
```

Comment:
or is this meant to hijack the messaging of that annoying "can't find log4j.properties" message that log4j provides